how-to/manage-environments/using-workspaces.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Using Workspaces
+description: Learn how to manage environments using Workspaces.
+---
+
+# Using Workspaces
+
+## Introduction
+
+Different environments for deployments are essential for gradually rolling out changes and testing new features in one environment (e.g., testing) before applying them to another environment (e.g. production). To achieve this, you must isolate your state files for each environment. In Terraform/OpenTofu, you can do this using Workspaces or File Layouts, each with pros and cons.
+
+## Workspaces
+
+Terraform Workspaces enable you to make multiple deployments using the same stack by isolating the state files for each deployment. This separation ensures that the state files for each deployment are stored in different locations, allowing the same infrastructure code to support multiple environments.
+
+When using different Terraform Workspaces, you can use the [`target`](../../cli/reference/cmdline/run.md#tmc-deployment-targets) option in Terramate to sync your deployments as distinct targets on Terramate Cloud. This feature treats each deployment with a different target as a separate entity on Terramate Cloud.
+
+## Steps
+
+1. Enable the experimental target feature by setting the project config option `terramate.config.experiments= ["targets"]`  before following the below steps.
+
+2. Create a new project, add a couple of stacks, and sync to the Terramate Cloud by following the quick-start guide up-to this [step](../../cli/getting-started/index.md#login-from-cli)
+
+3. List Available Terraform Workspaces in the bob stack:
+
+```bash
+cd stacks/bob
+
+terraform workspace list
+
+Sync the default workspace as the default target to Terramate Cloud:
+
+terramate run \               
+  --sync-drift-status \
+  --terraform-plan-file deploy.tfplan \
+  --target default \
```

Comment:
@AnnuCode when targets are enabled, does a user always need to explicitly set the default target or can it be omitted?

Reply:
We need to use `--target` when targets are enabled and when we are using any `--sync` option.